docs/src/api/params.md

```diff
@@ -514,6 +514,21 @@ Sets a consistent viewport for each page. Defaults to an 1280x720 viewport. `no_
 
 Does not enforce fixed viewport, allows resizing window in the headed mode.
 
+## context-option-clientCertificates
+- `clientCertificates` <[Array]<[Object]>>
+  - `url` <[string]> Glob pattern to match the URLs that the certificate is valid for.
+  - `certs` <[Array]<[Object]>> List of client certificates to be used.
+    - `cert` ?<[string]> Path to the file with the certificate in PEM format.
```

Comment:
Our goal is to make it CI friendly, so what about:

- If `pfx` first two bytes are `30 82` (magic bytes) then treat it as a PFX file - otherwise read it from the FS.
- If `cert` contains `-----BEGIN CERTIFICATE-----` ...
- If a `kek` contain `-----BEGIN PRIVATE KEY-----` or ` -----BEGIN RSA PRIVATE KEY-----`...

> Also, pfx is not a path, any specific reason it's different?

By accident.